DTXManiaプロジェクト/コード/ステージ/04.コンフィグ/CActConfigList.cs

```diff
@@ -2349,7 +2369,6 @@ private void tConfigIniへ記録する_Bass()
             //CDTXMania.ConfigIni.bLeft.Bass = this.iBassLeft.bON;
             //CDTXMania.ConfigIni.nInputAdjustTimeMs.Bass = this.iBassInputAdjustTimeMs.n現在の値;		// #23580 2011.1.3 yyagi
 
-            //CDTXMania.ConfigIni.b演奏音を強調する.Bass = this.iSystemSoundMonitorBass.bON;
```

Comment:
You'll see sound monitor bass/drums/guitar support being removed in various places, since it was unused by TJAP3 and had weird interactions with the existing volume control functions, making them only work in very specific and unexpected cases.